src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -127,27 +125,25 @@ describe('ATI Analytics Container', () => {
 
       renderer.create(
         <ContextWrap platform="amp" pageType="frontPage">
-          <ATIAnalytics data={mockData} />
+          <ATIAnalytics data={articleDataNews} />
         </ContextWrap>,
       );
 
       expect(mockAmp).toHaveBeenCalledTimes(1);
-      expect(mockAmp).toHaveBeenCalledWith(
-        {
-          pageviewParams: mockAtiQueryParams,
-        },
-        mockData,
-      );
+      expect(mockAmp.mock.calls[0][0]).toEqual({
+        pageviewParams: mockAtiQueryParams,
+      });
+
       expect(mockFrontPageAtiParams).toHaveBeenCalledTimes(1);
-      expect(mockFrontPageAtiParams).toHaveBeenCalledWith(mockData);
+      expect(mockFrontPageAtiParams).toHaveBeenCalledWith(articleDataNews);
     });
   });
 
   describe('pageType neither article nor frontPage', () => {
```

Comment:
if you wanted to have no warnings until https://github.com/bbc/simorgh/issues/3107 is done, you could
```suggestion
  // TODO fix this once bbc/simorgh#3107 has been completed
  describe.skip('pageType neither article nor frontPage', () => {
```